oleesir/diaryReload
# #160906406 user should be able to see the profile page

#### What does this PR do?
This pull request adds a profile template to the app

#### Description of Task to be completed?
* add profile template

#### How should this be manually tested?
* Navigate to the UI folder and open profile.html in your browser
* You will see a profile template

#### What are the relevant pivotal tracker stories?
[#160906406](https://www.pivotaltracker.com/story/show/160906406)

#### Any background context you want to add?
N/A

#### Screenshots
<img width="1280" alt="screen shot 2018-10-02 at 11 59 46 am" src="https://user-images.githubusercontent.com/17679511/46344967-be6cbc00-c63a-11e8-953a-c37c2a3cc809.png">
